### ISSUE TITLE
REST API Test gives '404 Not Found' error

### ISSUE THREAD

**Issue by limscentral** [OraOpenSource/oraclexe-apex#98]
My setup:

Cloud: Amazon AWS
Image / OS: CentOS 7 x86_64 (2014_09_29) (ami-c7d092f7)

Files:
oracle-xe-11.2.0-1.0.x86_64.rpm
apex_5.0.1.zip
ords.3.0.1.177.18.02.zip

Steps to recreate:
1) Run through installation procedure
2) Log into Internal Workspace and create a workspace
3) Log into new workspace
4) Go to SQL Workshop -> Restful Services -> Click on oracle.example.hr 
5) Choose Employees/ GET handler and click Test button (http://myserver.com/ords/WORKSPACENAME/hr/employees/)
6) Get 404 Error

Comments:  I never got prompted to enter anything in terms of ORDS configuration.  

![capture](https://cloud.githubusercontent.com/assets/13433479/8854902/f714993e-3127-11e5-9cb9-2dd12abbce67.PNG)

**Comment by limscentral** [comment]
I guess it's not a true issue since i used the ords version that is greater than the one listed.  But I wonder if the console unavailable would have caused an issue either way.  By the way, I used Putty from a windows client.  Not sure if that makes any difference.

**Comment by martindsouza** [comment]
@limscentral We haven't tested the scripts with 5.0.1 and the latest version of ORDS (though we did run through it with the initial version of 3). @tschf ran the scripts with APEX 5 and didn't have the issue. Given the error, did you update the `ords_params.properties` file with the appropriate password information (or modify the defaults)?

Using Putty to connect shouldn't have been an issue.

As for not being prompted by ORDS during install that is correct. We were able to remove that requirement to have a true silent install.
